hw/ip/sram_ctrl/doc/_index.md

```diff
@@ -103,35 +121,55 @@ This mechanism is part of the [life cycle]({{< relref "hw/ip/lc_ctrl/doc" >}}) s
 
 ## Scrambling Primitive
 
-The scrambling primitive `prim_ram_1p_scr` employs a reduced-round (5 instead of 11) PRINCE block cipher in CTR mode to scramble the data.
+The scrambling primitive [`prim_ram_1p_scr`]({{< relref "hw/ip/prim/doc/prim_ram_1p_scr" >}}) employs a reduced-round (5 instead of 11) PRINCE block cipher in CTR mode to scramble the data.
```

Comment:
is the text here a duplicate of the separate markdown?

Reply:
Part of it yes, and part of it has been slightly adapted to reflect the parameterization used here.
I can condense this down to reduce duplication.

Reply:
sounds good! yeah i think you'll be the best judge of what to duplicate.
I pointed it out mainly because when i have things in two places its super annoying to change them both when something is updated.

Reply:
OK I have condensed this section a bit and removed some of the low-level details of the scrambling primitive.